api/routes/cartRouter.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const { cartController } = require("../controllers")
+
+const router = express.Router();
+
+router.post('/add', cartController.addProductToCart);
```

Comment:
RESTful API 규칙에 맞게 end-point를 지정해 주세요.
REST에서는 end-point에 동사형을 사용하지 않습니다.
```suggestion
router.post('', cartController.addProductToCart);
```